tools/oesign/main.c

```diff
@@ -468,11 +480,20 @@ static const char _usage_gen[] =
 static const char _usage_sign[] =
     "Usage: %s sign {--enclave-image | -e} ENCLAVE_IMAGE "
     "{--config-file | -c} CONFIG_FILE {--key-file | -k} KEY_FILE\n"
+#if HAS_ENGINE_SUPPORT
+    "{--engine ENGINE-NAME --load-path ENGINE-LOAD-PATH --key-id KEY-ID }\n"
```

Comment:
Snake cased argument names for consistency with existing examples: `ENGINE_NAME`, `ENGINE_LOAD_PATH`, `KEY_ID` here and in the rest of the help text.

Reply:
done